### ISSUE TITLE
Support specifying custom endpoints paths

### ISSUE THREAD

**Issue by roskh** [axa-group/oauth2-mock-server#151]
## Summary

It would be useful to specify different paths for the endpoints other than the current ones since some OAuth2 clients
use hardcoded paths which can be different from the ones in this library (such as the Auth0 FE & BE clients)

## Desired solution

A way to specify the endpoints paths when starting server, e.g.:

```js
server.start(PORT, '0.0.0.0', {
  paths: {
    token_path: '/oauth/token'
  }
});
```

## Alternative solutions

N/A

## Additional Context

Auth0 first party client libraries use hard-coded paths for some OAuth2 endpoints, despite what the well know document might say, such as the token endpoint which is `/oauth/token` in Auth0 but just `/token` here.

**Comment by roskh** [comment]
I can take a stab at implementing this

**Comment by nulltoken** [comment]
Fixed by #152
